### PR TITLE
Add version bounds to torchvision / torchtext

### PR DESCRIPTION
**Pull request by sxjscience**
*Issue #, if available:*
- https://github.com/awslabs/autogluon/issues/1948
- https://github.com/awslabs/autogluon/issues/1944
- https://github.com/awslabs/autogluon/issues/1695

*Description of changes:*

Fix installation guide to include torchvision and torchtext. Also, add the version bound of torchvision/torchtext.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by zhiqiangdon — APPROVED** [review]
LGTM

File: docs/install-gpu-pip.rst [hunk 1]
```diff
@@ -3,4 +3,7 @@
     pip3 install -U pip
     pip3 install -U setuptools wheel
 
+    # Install the proper version of PyTorch following https://pytorch.org/get-started/locally/
+    pip3 install torch==1.12.0+cu113 torchvision==0.13.0+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
```
Comment (Innixma) [inline docs/install-gpu-pip.rst:7]: Why is this necessary? Wouldn't it automatically install this version anyways?

File: multimodal/setup.py [hunk 2]
```diff
@@ -28,6 +28,7 @@
     'requests',
     'timm<0.6.0',
     'torch>=1.9,<1.13',
+    'torchvision<0.14.0',
```
Comment (Innixma) [inline multimodal/setup.py:31]: I don't see the torchtext version bound mentioned in the PR title.

Comment (sxjscience) [reply multimodal/setup.py:31]: Verifying something on CoLAB. Do not merge for now.

Comment (sxjscience) [reply multimodal/setup.py:31]: I can confirm that `torchtext<0.14.0` is needed so I'm adding it back.

**Review by bryanyzhu — APPROVED** [review]

**Review by Innixma — APPROVED** [review]
